### ISSUE TITLE
Create pg_hba.conf from a file

### ISSUE THREAD

**Issue by jpninanjohn** [sorintlab/stolon#676]
### Submission type

 - Request for enhancement (RFE)

### Enhancement Description

For an existing cluster, is it possible to add a flag for the keeper to take in an existing pg_hba.conf file. If the file exists use it, if it does not, then do the default which is what it is doing now. 

Context:
We were trying to migrate an already running postgres to use stolon. The pg_hba.conf had many entries. Instead of recreating the entries by giving it as pgHBA in cluster spec, we thought it would be nice to have an option to give a file from which the pg_hba.conf can be created.

I am already working on a PR for this. Please do tell if there are any concerns.

**Comment by FarhanSajid1** [comment]
Any progress on this?

**Comment by jpninanjohn** [comment]
Yes. I have done it to a point. But have not worked on it for quite some time(around a year). Let me bring my fork up to date. Ill let you know.
